network/interlink/index.mdx

```diff
@@ -0,0 +1,55 @@
+---
+meta:
+  title: InterLink Documentation
+  description: Dive into Scaleway InterLink with our quickstart guides, how-tos, tutorials and more.
+  noindex: true
+---
+
+<Alert
+  sentiment="info"
+  title="InterLink is now in Private Beta"
+>
```

Comment:
Is this character ">" supposed to be here?

Reply:
Yes it's the end of the Alert tag